rust/remotivelabs-broker/examples/subscribe.rs

```diff
@@ -0,0 +1,74 @@
+use clap::Parser;
+use tonic::transport::Channel;
+
+use remotive_broker::{
+    check_license,
+    remotive_api::base::{
+        network_service_client::NetworkServiceClient, system_service_client::SystemServiceClient,
+        ClientId, NameSpace, SignalId, SignalIds, SubscriberConfig,
+    },
+};
+
+use std::vec::Vec;
+
+/// Rust subscribe example
+#[derive(Parser, Debug)]
+#[command(about)]
+struct Args {
+    /// Namespace
+    #[arg(short, long, required = true)]
+    namespace: String,
+
+    #[arg(short, long, required = true)]
+    signal: Vec<String>,
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let args = Args::parse();
+
+    let channel = Channel::from_static("http://localhost:50051")
```

Comment:
lets make broker URL a command line arg also

Reply:
Added. Also supports for X-API key.